src/app/main/home/detail/detail.component.ts

```diff
@@ -77,12 +79,16 @@ export class DetailComponent implements OnInit {
     const filter = {
       "ID": +this.templateId
     };
-    const columns = ["IMG", "TITLE", "DESCRIPTION", "TYPE", "IMAGES", "BOTTOM-DESCRIPTION"];
+    const columns = ["IMG", "TITLE", "DESCRIPTION", "TYPE", "IMAGES", "BOTTOM-DESCRIPTION", "URLGITHUB"];
     this.service.query(filter, columns, 'template').subscribe((response) => {
       if (response.code === 0 && Array.isArray(response.data)) {
         this.templateImg = response.data[0].IMG;
         this.templateTitle = response.data[0].TITLE;
         this.templateDescription = response.data[0].DESCRIPTION;
+        if (Util.isDefined(response.data[0].URLGITHUB)) {
+          this.templateURLReadme = '/ontimize-web-templates/develop/templates/' + response.data[0].URLGITHUB + '/README.md';
+          this.teplateURLGithub = 'https://github.com/OntimizeWeb/ontimize-web-templates/blob/develop/templates/' + response.data[0].URLGITHUB ;
+        }
```

Comment:
I think that it should be better to have all stuff related to Github inta an specific service for managing from there the url builders, constants, etc.
The urls should point to `main` shouldn't them?